pkg/cmd/admin/router/router.go

```diff
@@ -189,9 +189,10 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 		Ports:    defaultPorts,
 		Replicas: 1,
 
-		StatsUsername: "admin",
-		StatsPort:     defaultStatsPort,
-		HostNetwork:   true,
+		ServiceAccount: "router",
+		StatsUsername:  "admin",
+		StatsPort:      1936,
```

Comment:
Oops, nice catch.  I think I missed that when I merged in Ram's changes to enable healthz.  Fixed... PTAL?